src/ISOExtract.cpp

```diff
@@ -1104,6 +1121,10 @@ int ISOExtractClass::loadMDSTracks(const char *mdsFilename, FILE *isoFile, mds_s
 				}
 
 				fseek(isoFile, footer.filename_offset, SEEK_SET);
+
+            // Filenames should only be widechars on Windows;
```

Comment:
This is a horrible, horrible hack and I'm not proud of myself. This code doesn't compile on non-Windows systems because there isn't a widechar version of fopen on non-Windows systems. I'm sure this isn't the best way to organize this.

Reply:
The solution we came to with yabause was to reimplement the _wfopen function for ports that don't support it. Also we had to do some #define trickery for dealing with strcasecmp vs stricmp(since strcasecmp is posix only and stricmp is apparently windows only).